apps/comps/components/user-agents/index.tsx

```diff
@@ -113,9 +113,7 @@ export default function UserAgentsSection({ agents }: { agents: Agent[] }) {
           </Button>
         </div>
       </CollapsibleTrigger>
-      <CollapsibleContent className="w-full">
-        {agentList}
-      </CollapsibleContent>
+      <CollapsibleContent className="w-full">{agentList}</CollapsibleContent>
```

Comment:
there are a few diffs are due to the `format` rules being applied properly, but the GH workflow changes should now prevent this from happening.